django_ses_plus/models.py

```diff
@@ -32,12 +33,32 @@ def __str__(self):
         return f"{self.subject} to {self.to_email}"
 
 
+class SentEmailAttachment(models.Model):
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    sent_email = models.ForeignKey(to="django_ses_plus.SentEmail", related_name="attachments", on_delete=models.CASCADE)
+    name = models.CharField(max_length=255)
+    content = models.BinaryField()
+    type = models.CharField(max_length=255)
+
+    creation_datetime = models.DateTimeField(auto_now_add=True)
+    update_datetime = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        verbose_name = _("Sent Email Attachment")
+        verbose_name_plural = _("Sent Email Attachments")
+
+    def __str__(self):
+        return f"{self.name} ({self.type})"
+
+
 class SendEmailMixin(object):
 
     def get_to_email(self):
         return self.email
 
-    def send_email(self, subject, template_path, context, from_email=None, language=None):
+    def send_email(self, subject, template_path, context, attachments=None, from_email=None, language=None):
```

Comment:
What if someone used arguments like this

x.send_email("hi", "/template.html", {}, None, "TR")

I propose to move "attachments" to the end for backward compability.